contract/migrations/0009_auto_20210118_1427.py

```diff
@@ -0,0 +1,95 @@
+# Generated by Django 3.0.3 on 2021-01-18 14:27
+
```

Comment:
migration file related to the change in https://github.com/openimis/openimis-be-core_py/pull/36